### DISCUSSION TITLE
HMAC failed: Invalid Identifier Hash Provided

### DISCUSSION THREAD

**Discussion by drexlma** [chatwoot/chatwoot#6734] [Q&A]
Sometimes the error "HMAC failed: Invalid Identifier Hash Provided" appears. I can't find out when that occurs.
```
<?php

$key = 'xxx';
$message = '620';

echo $chat_identifier_hash = hash_hmac('sha256', $message, $key);
```

When I connect to a new computer it works. its only on specific computers. 
Maybe if you have already written to another inbox (chat)?

**Comment by pranavrajs** [comment]
@drexlma This shouldn't be happening even if you have another another conversation. Moving this to discussions at the moment. Please let us know when you have an case where you can reproduce this consistently, we will raise the bug request then.
